DigitalOffice/Services/CompanyService/Repositories/CompanyRepository.cs

```diff
@@ -0,0 +1,40 @@
+﻿using LT.DigitalOffice.CompanyService.Database;
+using LT.DigitalOffice.CompanyService.Models;
+using LT.DigitalOffice.CompanyService.Repositories.Interfaces;
+using System;
+using System.Linq;
+
+namespace LT.DigitalOffice.CompanyService.Repositories
+{
+    public class CompanyRepository : ICompanyRepository
+    {
+        private readonly CompanyServiceDbContext dbContext;
+
+        public CompanyRepository(CompanyServiceDbContext dbContext)
+        {
+            this.dbContext = dbContext;
+        }
+
+        public bool UpdateCompany(ChangeCompanyRequest company)
+        {
+            var dbCompany = dbContext.Companies.FirstOrDefault(x => x.Id == company.CompanyId);
+            if (dbCompany == null)
+            {
+                throw new Exception("Company was not found.");
+            }
+
+            if (company.Name != null)
+            {
+                dbCompany.Name = company.Name;
+            }
+            if (company.IsActive != null)
+            {
+                var isActive = (bool)company.IsActive;
+                dbCompany.IsActive = isActive;
+            }
+            dbContext.SaveChanges();
+
+            return true;
```

Comment:
Pavel, have you checked this in an actual Database?
I think this logic is broken. You just set object's properties, not the data in the actual database.

To update the DbEntity, you should use .Update(entity) method. Please, look it up. You'll get my point.

Reply:
Here's how I've tested it in my PR: 
![image](https://user-images.githubusercontent.com/61093487/89717847-f07f9f80-d9c2-11ea-88e6-d917827c4299.png)

Reply:
Not relevant.